crates/ruff_python_formatter/src/lib.rs

```diff
@@ -84,16 +91,40 @@ where
     }
 }
 
-pub fn format_module(contents: &str, options: PyFormatOptions) -> Result<Printed> {
+#[derive(Error, Debug)]
+pub enum FormatModuleError {
```

Comment:
the `anyhow::Result` from `format_node` had irked me already and now i had the motivation to change it